bam_qc_metrics/bam_qc.py

```diff
@@ -727,8 +812,69 @@ def run_samtools_stats(self, args, strict=False):
             else:
                 self.logger.warning(msg)
         return stats_str
-    
-class slow_metric_finder(base):
+
+class fast_metric_writer(base):
+    """
+    Wrapper for the fast_metric_finder class, to read parameters, set up logging, and write JSON.
+    Enables fast metrics to be computed as a standalone action (eg. for RNASeqQC).
+    """
+
+    def __init__(self, config):
+        self.validate_config_fields(config)
+        # set up logging
+        self.debug = config[self.CONFIG_KEY_DEBUG]
+        self.verbose = config[self.CONFIG_KEY_VERBOSE]
```

Comment:
`self.debug` and `self.verbose` are not used.

Reply:
They are used by the following call to self.configure_logger. Although, as you rightly point out, that method needs to be modified so the base class doesn't impllicitly assume self.debug and self.verbose exist.

Reply:
`debug` and `verbose` are no longer instance variables, instead they are simply arguments to `configure_logger()`